### ISSUE TITLE
VS 2017 RC "Edit WCF Configuration" Windows SDK installation error

### ISSUE THREAD

**Issue by scottaddie** [aspnet/Tooling#954]
Create a new ASP.NET Core 1.1 Web Application targeting .NET Core using the project template. Right-click the `web.config` file in Solution Explorer, and select **Edit WCF Configuration** from the context menu. The following error appears:

![windows_sdk_error](https://cloud.githubusercontent.com/assets/10702007/22044869/bf6f6030-dccb-11e6-8e4d-2cc1ee32018a.png)

**Comment by mlorbetske** [comment]
TFS#369185

**Comment by dasetser** [comment]
@scottaddie Thanks for reporting this.

Why do you want to edit the WCF Configuration of the web.config when you're using ASP.NET Core? I believe that web.config is only used to turn on the AspNetCoreModule in IIS. WCF clients using .NET core won't read from this config file, so I'm not aware of any scenario where you would need to do this.

I do see that the error message is unhelpful (and incorrect). We can look into giving an accurate error message or maybe removing this from the context menu, but I'm not sure why this would need to be supported.

**Comment by scottaddie** [comment]
@dasetser I was very surprised to see **Edit WCF Configuration** in the context menu. My recommendation would be to remove it.

**Comment by dasetser** [comment]
I agree that's the best way to handle it. I just wanted to confirm with you that you didn't have a scenario you expected this to be useful for. We'll look into removing it in a future release.
